hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkAdminHandlerTestBase.java

```diff
@@ -402,15 +446,15 @@ protected void givenNonFungibleTokenRelation() {
                 .build();
     }
 
-    private TransactionID transactionID(int nonce) {
-        return transactionID(0, nonce);
+    private TransactionID transactionID(Instant validStartTime, int nonce) {
+        return transactionID(validStartTime, 0, nonce);
```

Comment:
Why not use the nanos in the Instant?
```suggestion
        return transactionID(validStartTime, validStartTime.getNano(), nonce);
```
It is a bit odd that the base overload separates them out to begin with, but at least here we can simplify that.